internal/warehouse/githubActions.go

```diff
@@ -0,0 +1,416 @@
+package warehouse
+
```

Comment:
can we rename this file `github_actions.go` to be consistent with go file naming (and how we do it elsewhere in the repo)

Reply:
done